.circleci/ci-tests/terraform-generated-aws/service-http.yaml

```diff
@@ -0,0 +1,20 @@
+environments:
+  - name: env-tg
+    path: "environment.yaml"
+    variables:
+      max_containers: 2
+name: http-service
+modules:
+  - name: app
+    type: k8s-service
+    image: kennethreitz/httpbin
```

Comment:
note: getting started use `ghcr.io/run-x/opta-examples/hello-app:main` which is very similar but not subject to `[docker hub rate limit](https://docs.docker.com/docker-hub/download-rate-limit/)`, which we might eventually hit.

Reply:
I don't understand what you mean by that. 
Are you in favour of using the `kennethreitz/httpbin` image or our own Hello world app.
As for the pull rate limit, it's 200 per 6 hrs, so that shouldn't be of any worry to us.

Reply:
yeah not a big deal, but if we use this image in all our tests and we run test for every merge we might eventually hit this limit.

Reply:
I am using this image because this is a standard image we use for testing all things internal, hence chose to use this.

Reply:
this is fine for this PR, but may be we should start using the new image with new tests, to use the same image that is used in our docs (dogfooding)